pkg/agent/handler/accesscontrollist.go

```diff
@@ -22,7 +22,7 @@ func NewACLHandler(agentCacheManager agentcache.Manager) Handler {
 
 // Handle processes grpc events triggered for AccessRequests
 func (h *aclHandler) Handle(ctx context.Context, meta *proto.EventMeta, resource *v1.ResourceInstance) error {
-	action := getActionFromContext(ctx)
+	action := GetActionFromContext(ctx)
```

Comment:
How come this is being exported now?

Reply:
Governance agent is to register handler outside of the Agent SDK.